### ISSUE TITLE
Right click on groups => Uncaught TypeError: Cannot read properties of undefined (reading 'length')

### ISSUE THREAD

**Issue by var1ableX** [comfyanonymous/ComfyUI#4927]
### Update:
I had --front-end-version set to latest on start and this seems to be an issue in 1.2.53, I tried 1.2.48 and problem went away. Removing the --front-end-version flag also fixes the issue. Feel free to close this issue if it's not useful.

### Expected Behavior

Context menu should appear when right clicking on a group

### Actual Behavior

UI unresponsive

### Steps to Reproduce

1. Start comfyui
2. Load default workflow
3. Select all nodes
4. Right click on canvas and select: Add group for selected nodes
5. Right click on group

### Debug Logs

```powershell
groupOptions.ts:105 Uncaught TypeError: Cannot read properties of undefined (reading 'length')
    at LGraphCanvas.getCanvasMenuOptions (groupOptions.ts:105:24)
    at LGraphCanvas.getCanvasMenuOptions (nodeTemplates.ts:364:28)
    at LGraphCanvas.getCanvasMenuOptions (graphArrange.js:8:25)
    at LGraphCanvas.getCanvasMenuOptions (nodeFinder.js:25:25)
    at LGraphCanvas.getCanvasMenuOptions (use_everywhere.js:209:59)
    at LGraphCanvas.getCanvasMenuOptions (menu_queue_node.js:63:50)
    at LGraphCanvas.getCanvasMenuOptions (workflowImage.js:594:26)
    at LGraphCanvas.getCanvasMenuOptions (ui_mixlab.js:1886:30)
    at __LGraphCanvas.processContextMenu (litegraph.es.js:10987:28)
    at __LGraphCanvas.processMouseDown (litegraph.es.js:6100:16)
LGraphCanvas.getCanvasMenuOptions @ groupOptions.ts:105
LGraphCanvas.getCanvasMenuOptions @ nodeTemplates.ts:364
LGraphCanvas.getCanvasMenuOptions @ graphArrange.js:8
LGraphCanvas.getCanvasMenuOptions @ nodeFinder.js:25
LGraphCanvas.getCanvasMenuOptions @ use_everywhere.js:209
LGraphCanvas.getCanvasMenuOptions @ menu_queue_node.js:63
LGraphCanvas.getCanvasMenuOptions @ workflowImage.js:594
LGraphCanvas.getCanvasMenuOptions @ ui_mixlab.js:1886
processContextMenu @ litegraph.es.js:10987
processMouseDown @ litegraph.es.js:6100
LGraphCanvas.processMouseDown @ changeTracker.ts:179
LGraphCanvas.processMouseDown @ simpleTouchSupport.ts:102
LGraphCanvas.processMouseDown @ rgthree.js:224
LGraphCanvas.processMouseDown @ app.ts:1192
```

**Comment by Elgokoo** [comment]
me too, the right click on group doesn't work

**Comment by ltdrdata** [comment]
Update rgthree and try again. If you still encounter the same error, try disabling it and then retry.

**Comment by LukeG89** [comment]
The issue is fixed in version 1.2.55 after this commit: https://github.com/Comfy-Org/ComfyUI_frontend/commit/429fa75fccbfbaa6ce97c6d5efa5780bac15a44f

Just update the Front-End
